namespace-specific-policy/automated-rendering/README.md

```diff
@@ -19,13 +19,11 @@ This section describes prerequisites you must meet before this tutorial.
 
 ### Get the example configuration
 
-The `automated-rendering` directory includes a root `kustomization.yaml` file that references the three team overlays.
+The `automated-rendering` directory includes a root `kustomization.yaml` file that references the `example` kustomization directory.
```

Comment:
references three team overlays in the `example` kustomization directory, and an `external-team` overlay.

Reply:
Done